<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EULAForm</class>
 <widget class="QWidget" name="EULAForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terms and Conditions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>251</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblVersion">
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblVersionValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblAppName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="txtEula"/>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGraphicsView" name="gphView"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkAccept">
     <property name="text">
      <string>I accept the &quot;Terms and Conditions&quot;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bConfirm">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkAccept</tabstop>
  <tabstop>bConfirm</tabstop>
  <tabstop>bExit</tabstop>
  <tabstop>gphView</tabstop>
  <tabstop>txtEula</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bExit</sender>
   <signal>clicked()</signal>
   <receiver>EULAForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
